manifests/init.pp

```diff
@@ -64,7 +73,10 @@
   $package_name            = $nsclient::params::package_name,
   $download_destination    = $nsclient::params::download_destination,
   $config_template         = $nsclient::params::config_template,
-  $install_path            = $nsclient::params::install_path
+  $install_path            = $nsclient::params::install_path,
+  Boolean $chocolatey_provider                    =  $nsclient::params::chocolatey_provider,
```

Comment:
sorry, I was a bit unclear. Can you please align all `=` chars in this params block?